bindings/experimental/distrdf/python/DistRDF/__init__.py

```diff
@@ -63,6 +65,55 @@ def create_logger(level="WARNING", log_path="./DistRDF.log"):
     return logger
 
 
+def RunGraphs(proxies):
```

Comment:
Perhaps we could think of a better place where to define `RunGraphs`? In its own module? Maybe we could have a module called e.g. `API.py` at this level where we define all the calls in `distributed`?
Can we list what functions would go there? `initialize`? Where are the `include_headers`, `include_libraries`?

Reply:
We can just merge this and then discuss the refactoring later.

Reply:
Yes this is needed at some point. Especially for the header functionality, which is currently a bit hidden in the backend object